### PR TITLE
update to babel7.0 and other deps updates

### PR DESCRIPTION
**Pull request by digitalkaoz**
EOM

### CODE REVIEW

**Comment by bkniffler** [comment]
If this could be merged and published, I'd be very happy!

**Comment by nozzle** [comment]
Info: `master` branch now points to 5.x.x and the `v6` branch to 6.0.0-beta. You may need to close and reopen your PR to the v6 branch.

File: package.json [hunk 1]
```diff
@@ -114,16 +114,16 @@
     "enzyme": "^3.4.3",
     "enzyme-adapter-react-16": "^1.2.0",
     "enzyme-to-json": "^3.3.4",
-    "eslint": "4",
-    "eslint-config-prettier": "^2.9.0",
+    "eslint": "^5.4.0",
```
Comment (tannerlinsley) [inline package.json]: Last I checked, this version of eslint still had issues. I think we should hold off on this one for a bit more... Thoughts?

Comment (digitalkaoz) [reply package.json]: i expected to issues. do you have any links?

**Comment by tannerlinsley** [comment]
No but I remember there being an incompatibility with the Airbnb config.

**Comment by digitalkaoz** [comment]
worked correctly while crafting this PR, so i dont know

**Comment by tannerlinsley** [comment]
We'll go with it then. If it pops up again, we can handle it easily.

**Pull request merged**